src/reference/00-Getting-Started/05-Basic-Def.md

```diff
@@ -2,156 +2,129 @@
 out: Basic-Def.html
 ---
 
-  [More-About-Settings]: More-About-Settings.html
+  [Task-Graph]: Task-Graph.html
   [Bare-Def]: Bare-Def.html
   [Full-Def]: Full-Def.html
   [Running]: Running.html
   [Library-Dependencies]: Library-Dependencies.html
   [Input-Tasks]: ../docs/Input-Tasks.html
 
-.sbt build definition
----------------------
+Build definition
+----------------
 
 This page describes sbt build definitions, including some "theory" and
-the syntax of `build.sbt`. It assumes you know how to [use sbt][Running]
+the syntax of `build.sbt`.
+It assumes you have installed a recent version of sbt, such as sbt 0.13.13,
+know how to [use sbt][Running],
 and have read the previous pages in the Getting Started Guide.
 
-### Three Flavors of Build Definition
+This page discusses the `build.sbt` build definition.
 
-There are three flavors of build definition:
+### Specifying the sbt version
 
-1. Multi-project `.sbt` build definition
-2. Bare `.sbt` build definition
-3. `.scala` build definition
+As part of your build definition you will specify the version of
+sbt that your build uses.
+This allows people with different versions of the sbt launcher to
+build the same projects with consistent results.
+To do this, create a file named `project/build.properties` that specifies the sbt version as follows:
 
-This page discusses the newest multi-project `.sbt` build definition, which combines the strength
-of the two older flavors, and is suitable for all cases.
-You might come across the other older flavors when dealing with builds in the wild.
-See [bare .sbt build definition][Bare-Def] and [.scala build definition][Full-Def] (later in Getting Started) for more
-on other flavors.
+```
+sbt.version=$app_version$
+```
 
-In addition, a build definition can contain files ending in `.scala`, located in the
-`project/` subdirectory of the base directory to define commonly used functions and values.
+If the required version is not available locally,
+the `sbt` launcher will download it for you.
+If this file is not present, the `sbt` launcher will choose an arbitrary version,
+which is discouraged because it makes your build non-portable.
 
-### What is a Build Definition?
+### What is a build definition?
 
-After examining a set of directories and processing build definition files, sbt
-ends up with `Project` definitions.
+A *build definition* is defined in `build.sbt`,
+and it consists of a set of projects (of type [`Project`](../api/sbt/Project.html)).
+Because the term *project* can be ambiguous,
+we often call it a *subproject* in this guide.
 
-In `build.sbt` you might create a [Project](../api/sbt/Project.html) definition of
-the project located in the current directory like this:
+For instance, in `build.sbt` you define
+the subproject located in the current directory like this:
 
 ```scala
 lazy val root = (project in file("."))
+  .settings(
```

Comment:
There is a page called [Appendix: Bare .sbt build definition](http://www.scala-sbt.org/0.13/docs/Bare-Def.html), but I'm steering new users away from using that in build.sbt.